com.woltlab.wcf/templates/membersList.tpl

```diff
@@ -134,13 +134,10 @@
 	$(function() {
 		WCF.Language.addObject({
 			'wcf.user.button.follow': '{jslang}wcf.user.button.follow{/jslang}',
-			'wcf.user.button.ignore': '{jslang}wcf.user.button.ignore{/jslang}',
 			'wcf.user.button.unfollow': '{jslang}wcf.user.button.unfollow{/jslang}',
-			'wcf.user.button.unignore': '{jslang}wcf.user.button.unignore{/jslang}'
 		});
 		
 		new WCF.User.Action.Follow($('.userList > li'));
-		new WCF.User.Action.Ignore($('.userList > li'));
```

Comment:
Why is ignoring no longer possible on list pages?

Reply:
See the commit message here: https://github.com/WoltLab/WCF/pull/4064/commits/27eea3e631312f8111d48550eb342902d00fbe5e